deploy/kubernetes/base/controller/controller.yaml

```diff
@@ -29,13 +29,15 @@ spec:
             - "--csi-address=/csi/csi.sock"
             - "--feature-gates=Topology=true"
             - "--metrics-address=:22011"
-            - "--enable-leader-election"
-            - "--leader-election-type=leases"
+            # - "--enable-leader-election"      # remove deprecated leader election flag
```

Comment:
remove these lines?

Reply:
Done. Slightly concerned about breaking an overlay that might reference arg by index number, but hopefully we have enough tests to catch that.

Reply:
We should have removed all overlays that patch by arg number?

Reply:
Should be.